spec/requests/authentication_spec.rb

```diff
@@ -4,6 +4,10 @@
 describe "Authentication API" do
   ENTRYPOINT_KEYS = %w(name description version versions identity collections)
 
+  # some tests change token_ttl that affects authorization tokens.
+  before { Api::Environment.clear_caches }
```

Comment:
These should probably be cleared on every test - the caller shouldn't have to know that `clear_caches` happens to clear the token_ttl.

That being said, I'm fine with this for now to fix these issues.